text.go

```diff
@@ -65,6 +65,7 @@ type Text struct {
 	q1      int
 	what    TextKind
 	tabstop int
+	texpand bool
```

Comment:
I'd like this to be called tabexpand.

Reply:
Done. Tab expansion isn't persisted in the dump file, but if this is desired I can add that as well. I should have some time this weekend to add tests for these features.